pkg/resource/check.go

```diff
@@ -6,10 +6,11 @@ package resource
 import (
 	"context"
 	"fmt"
```

Comment:
```suggestion
	"fmt"
	"slices"

```

I think we are missing the goimports linter in this repo but lets make it nice for when we add it.